platforms/android/library/src/main/java/io/element/android/wysiwyg/inputhandlers/InterceptInputConnection.kt

```diff
@@ -322,7 +323,12 @@ internal class InterceptInputConnection(
         beginBatchEdit()
         editable.removeFormattingSpans()
         editable.replace(0, editable.length, charSequence)
-        setComposingRegion(compositionStart, compositionEnd)
+        val start = compositionStart.coerceIn(0, editable.length)
+        val end = compositionEnd.coerceIn(0, editable.length)
+        val newComposition = editable.substring(start, end)
+        if (newComposition.isEmpty() || !newComposition.isDigitsOnly()) {
```

Comment:
It seems a bit odd that `replaceAll` will not honour the given `compositionStart` and `compositionEnd`. Maybe the caller could be responsible for passing the right arguments? Or we can move the responsibility of setting the new composition out of `replaceAll`?

Reply:
I changed this on https://github.com/matrix-org/matrix-rich-text-editor/pull/878/commits/88c81f918230f1c18295795ba23b23bd3e56c898